contracts/tokenomics/generator/src/state.rs

```diff
@@ -5,50 +5,67 @@ use cw_storage_plus::{Item, Map};
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-// Info of each user.
+/// ## Description
+/// This structure describes the main information of each user
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema, Default)]
 pub struct UserInfo {
+    /// Sets an amount
     pub amount: Uint128,
+    /// Sets a reward debt TODO:
```

Comment:
A reward amount user already received or is not eligible for, used for proper reward calculation